index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Web Calculator</title>
+  </head>
+
+  <body>
+    <h1>Calculator</h1>
+    <form id="myForm">
+      <input type="text" id="num1" placeholder="first number" required />
+
+      <select id="operator">
+        <option value="add">+</option>
+        <option value="sub">-</option>
+        <option value="mult">*</option>
+        <option value="div">/</option>
+      </select>
+
+      <input type="text" id="num2" placeholder="second number" required />
+
+      <input type="submit" value="Calculate" />
+    </form>
+
+    <h2 id="result">Answer is:</h2>
+
+    <script type="text/javascript" src="wcpart2.js"></script>
```

Comment:
Don't include your script tag down here. The only reason you would want to do that is for very large script files that are causing your page to load slowly. Put it in your head section.